piksi_multi_cpp/src/sbp_callback_handler/sbp_callback_handler_relay/ros_enu_relays.cc

```diff
@@ -1,27 +1,22 @@
-#include <eigen_conversions/eigen_msg.h>
-#include <libsbp_ros_msgs/ros_conversion.h>
-#include <ros/assert.h>
 #include "piksi_multi_cpp/sbp_callback_handler/sbp_callback_handler_relay/ros_enu_relays.h"
 
 namespace piksi_multi_cpp {
 
-namespace lrm = libsbp_ros_msgs;
-
 bool RosPosEnuRelay::convertSbpMsgToRosMsg(const msg_pos_ecef_t& in,
                                            const uint8_t len,
                                            geometry_msgs::PointStamped* out) {
   ROS_ASSERT(out);
 
-  // Convert position.
-  Eigen::Vector3d x_ecef, x_enu;
-  lrm::convertCartesianPoint<msg_pos_ecef_t>(in, &x_ecef);
+  return convertEcefToEnu(in, &out->point);
+}
 
-  if (!geotf_handler_.get()) return false;
-  if (!geotf_handler_->getGeoTf().convert("ecef", x_ecef, "enu", &x_enu))
-    return false;
+bool RosTransformEnuRelay::convertSbpMsgToRosMsg(
+    const msg_pos_ecef_t& in, const uint8_t len,
+    geometry_msgs::TransformStamped* out) {
+  ROS_ASSERT(out);
 
-  tf::pointEigenToMsg(x_enu, out->point);
-  return true;
+  // TODO(rikba): Also add orientation information if available.
+  return convertEcefToEnu(in, &out->transform.translation);
```

Comment:
I know that this is kind of hacky, but even if we don't use the orientation it would be good if we would still set the orientation to (0,0,0,1), otherwise rviz keeps complaining about and invalid transform.